root/etc/e-smith/templates/etc/ssh/sshd_config/50SubsystemSftp

```diff
@@ -5,7 +5,9 @@
 
     my $SubsystemSftp = $sshd{"SubsystemSftp"} || "yes";
 
-    $OUT = ($SubsystemSftp eq "yes") ? 
-	"Subsystem      sftp    /usr/libexec/openssh/sftp-server" : 
-	"# The sftp Subsystem is disabled in the configuration database";
+    if ($SubsystemSftp eq "yes") {
+      $OUT .= "Subsystem sftp internal-sftp -u 0077\n";
```

Comment:
Why the umask is now explicit? Why that value?

Reply:
@DavidePrincipi proposed a umask, but less restrictive...only user can read/modify files in their home 

:-?

Reply:
ok